operator/watchers/pod.go

```diff
@@ -42,19 +41,13 @@ var (
 	UnmanagedPodStoreSynced = make(chan struct{})
 )
 
-// podNodeNameIndexFunc indexes pods by node name
-func podNodeNameIndexFunc(obj interface{}) ([]string, error) {
-	pod := obj.(*slim_corev1.Pod)
-	if pod.Spec.NodeName != "" {
-		return []string{pod.Spec.NodeName}, nil
-	}
-	return []string{}, nil
-}
-
+// PodsInit and PodStore will be deprecated with other watchers, once all of
+// their uses move to hive/cell model that uses k8s resources instead of
+// separately initializing informers.
 func PodsInit(ctx context.Context, wg *sync.WaitGroup, clientset k8sClient.Clientset) {
```

Comment:
I think PodsInit is unused, we could just remove it.

Reply:
Sounds good. I'll do it as a follow up.